python_otbr_api/__init__.py

```diff
@@ -20,6 +26,18 @@ def __init__(
         self._url = url
         self._timeout = timeout
 
+    async def set_enabled(self, enabled: bool) -> None:
+        """Enable or disable the router."""
+
+        response = await self._session.post(
+            f"{self._url}/node/state",
+            json='"enabled"' if enabled else '"disabled"',
```

Comment:
anything you pass to `json` will be JSON stringified. So that would mean these values become `"\"enabled\""` is that correct?

Reply:
We should indeed just send a JSON encoded string, fixed now 👍